core/server/api/v2/utils/serializers/input/tags.js

```diff
@@ -0,0 +1,9 @@
+const debug = require('ghost-ignition').debug('api:v2:utils:serializers:input:tags');
+const url = require('./utils/url');
+
+module.exports = {
+    edit(apiConfig, frame) {
```

Comment:
shouldn't the `add` method be covered here as well?

Reply:
Maybe :man_shrugging:  following a similar pattern as in posts input serialize would be good, e.g.:
```
    edit(apiConfig, frame) {
        this.add(apiConfig, frame);
    }
```

Reply:
Absolute URLs are only sent by Ghost-Admin after the initial save happens, before that it sends relative URL first time, so we don't have the problem there. I tried cases if its possible to have absolute url on add, and wasn't able to replicate, so only added `edit`. Made exception in post as its `edit` was anyways calling `add` directly so added directly in there :)

Reply:
We shouldn't be making any server-side changes based solely on what Ghost-Admin does. If we're modifying input values for one type of request we should be modifying them for all requests.

Reply:
Agreed, but we should be always thinking about the general use case as Ghost-Admin might not be the only client in the future that uses the API :wink:  Maybe adding these cases in our functional test would make things more clean and explicit?

Reply:
> We shouldn't be making any server-side changes based solely on what Ghost-Admin does.

😄 Good point, updated.

Reply:
> Maybe adding these cases in our functional test would make things more clean and explicit?

🤔 We have unit tests for input serializers for this case now, should we also add to functional test as well?